maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -568,9 +568,16 @@ class AndroidDriver(
         LOGGER.info("[Done] Adding media files")
     }
 
-    fun setDeviceLocale(country: String, language: String) {
+    fun setDeviceLocale(country: String, language: String): Int {
         dadb.shell("pm grant dev.mobile.maestro android.permission.CHANGE_CONFIGURATION")
-        dadb.shell("am broadcast -a dev.mobile.maestro.locale -n dev.mobile.maestro/.receivers.LocaleSettingReceiver --es lang $language --es country $country")
+        val response = dadb.shell("am broadcast -a dev.mobile.maestro.locale -n dev.mobile.maestro/.receivers.LocaleSettingReceiver --es lang $language --es country $country")
```

Comment:
How will we surface any error thrown by this call?

Reply:
this call is always successful , exceptions happen on android app level, handled them by propagating certain error codes and data messages for now